modules.d/95nvmf/nvmf-autoconnect.sh

```diff
@@ -1,5 +1,54 @@
-#!/bin/bash
+#!/bin/sh
+# Argument $1 is "settled", "online", or "timeout", indicating
```

Comment:
Only "timeout" is taking into account, the other parameters are only for debugging purposes, right?

Reply:
Yes. The argument allows the script to figure out the type of event it was being called for. Currently "timeout" is the only event type that gets a special treatment.

I realize that the value "online" actually doesn't occur any more, because we're using the netroot handler for that purpose. I'll remove it from the comment to avoid confusion.